src/engraving/types/typesconv.cpp

```diff
@@ -242,8 +242,8 @@ static const std::vector<Item<ElementType> > ELEMENT_TYPES = {
     { ElementType::COMPOUND,             "Compound",             QT_TRANSLATE_NOOP("engraving", "Compound") },
     { ElementType::CHORD,                "Chord",                QT_TRANSLATE_NOOP("engraving", "Chord") },
     { ElementType::SLUR,                 "Slur",                 QT_TRANSLATE_NOOP("engraving", "Slur") },
-    { ElementType::ELEMENT,              "EngravingItem",        QT_TRANSLATE_NOOP("engraving", "EngravingItem") },
-    { ElementType::ELEMENT_LIST,         "ElementList",          QT_TRANSLATE_NOOP("engraving", "EngravingItem list") },
+    { ElementType::ELEMENT,              "EngravingItem",        QT_TRANSLATE_NOOP("engraving", "Engraving item") },
+    { ElementType::ELEMENT_LIST,         "ElementList",          QT_TRANSLATE_NOOP("engraving", "Engraving item list") },
```

Comment:
I doubt about these ones; in other places in the UI, we still call it "Elements", for example "select similar elements", and I think "Element" is better as a user-exposed string than "Engraving item". I'd vote for reverting the usernames to "Element".

Reply:
ok, done